_includes/life_cycle.html

```diff
@@ -28,7 +28,55 @@
 {% elsif site.life_cycle == "stable" %}
 
 {% comment %}
-We do not do anything special unless this is a Carpentries Lab lesson
+We do not do anything special for lessons in stable
```

Comment:
For lessons in stable... what?

Reply:
"stable" here refers to the last stage in [the lesson life cycle described in The Carpentries Curriculum Development Handbook](https://cdh.carpentries.org/the-lesson-life-cycle.html). This stage is used when the lesson is considered to be fully released i.e. out of beta testing.